build.gradle

```diff
@@ -69,6 +93,15 @@ configure(subprojects - project(":tests:gdx-tests-android")) {
         options.fork = true
         options.incremental = true
     }
+    
+    java {
+        /*withJavadocJar()*/
+        withSourcesJar()
+    }
+    
+    javadoc {
+        options.addStringOption('Xdoclint:none', '-quiet')
```

Comment:
I guess we're requiring Java 8 to build now! 🎉 From what I can tell (given that the Java 7 man pages are no longer available on Oracle's site), the doclint option for javadoc was added in Java 8.

In my projects, I use the code:
```groovy
if (JavaVersion.current().isJava8Compatible()) {
    allprojects {
        tasks.withType(Javadoc) {
            options.addStringOption('Xdoclint:none,-missing', '-quiet')
        }
    }
}
```

Reply:
Ah whoops. I didn't realize that was Java 8 specific.

It's probably time to move to Java 8 to compile, but for now I'll add this safety check.

Thanks!

Reply:
Looks like you got this back to compatible with Java 7; I'll resolve this.